bin/game.rb

```diff
@@ -0,0 +1,48 @@
+# Class Game is the main class
+class Game
+  require '../lib/board'
+  require '../lib/message'
+
+  def initialize
+    @turn = 1
+    @winner = false
+    @game_over = false
+
+    @board = Board.new
+    move
+  end
+
+  def move
+    @board.show_board
+    if @game_over
+      if @winner
+        Message.new(:winner, @turn)
+      else
+        Message.new(:tie)
+      end
+    else
+      Message.new(:instruction)
+      Message.new(:turn, @turn)
+      field = gets.chomp
+      play(field)
+    end
+  end
+
+  def play(field)
+    if field.length == 2 && "012".include?(field[0]) && "012".include?(field[1])
```

Comment:
The `play` method is sanitizing the input. I don't think that is the responsibility of this method. What about having a method that gets the user input and then sanitizes it so that we are sure we are getting a correct input?